src/ui/src/components/recordTable/recordTable.tsx

```diff
@@ -0,0 +1,102 @@
+import { Table } from "@trussworks/react-uswds";
+import { Record } from "@/models/record";
+import Link from "next/link";
+import { JSX } from "react";
+
+export interface RecordTableProps {
+  items: Record[];
+  withReviewLink?: boolean;
+}
+
+function getLinkScoreEl(linkScore: number): JSX.Element {
+  if (linkScore > 0.9) {
+    return (
+      <>
+        <span className="text-semibold text-good">{linkScore}</span>{" "}
+        <span className="usa-sr-only">good</span>
+      </>
+    );
+  } else if (linkScore > 0.8) {
+    return (
+      <>
+        <span className="text-semibold text-okay">{linkScore}</span>{" "}
+        <span className="usa-sr-only">okay</span>
+      </>
+    );
+  } else {
+    return (
+      <>
+        <span className="text-semibold text-bad">{linkScore}</span>{" "}
+        <span className="usa-sr-only">bad</span>
+      </>
+    );
+  }
+}
+
+function getTableRow(record: Record, withReviewLink: boolean): JSX.Element {
+  return (
+    <tr key={record.id}>
+      <td>
+        <span className="text-bold">
+          {record.patient.LastName}, {record.patient.firstName}
+        </span>
+        <br />
+        <span className="text-base">
+          DOB: {record.patient.dob.toLocaleDateString()}
+        </span>
+      </td>
+      <td>
+        <span>{record.receivedOn.toLocaleDateString()}</span>
+        <br />
+        <span className="text-base">
+          {record.receivedOn.toLocaleTimeString([], {
+            hour: "numeric",
+            minute: "2-digit",
+            hour12: true,
+          })}
+        </span>
+      </td>
+      <td>
+        <span>{record.dataStream.name}</span>
```

Comment:
```suggestion
        <span>{record.dataStream.system}</span>
```

Reply:
Same as comment below about this PR not integrating the real data structures.